packages/cashier/src/Components/Form/crypto-withdraw-form.jsx

```diff
@@ -87,13 +87,14 @@ const CryptoWithdrawForm = ({
 
     return (
         <div className='cashier__wrapper'>
-            {!isMobile() && <Header currency={currency} />}
-            <Icon
-                icon={`IcCurrency-${account_platform_icon.toLowerCase()}`}
-                size={isMobile() ? 64 : 128}
-                className='crypto-withdraw-form__icon'
-            />
-            {isMobile() && <Header currency={currency} />}
+            <Header currency={currency} />
```

Comment:
@likhith-deriv Why did you omit the `isMobile` condition for showing the header?
Do we have any design changes?
Base on the old code the header should show after the icon in mobile view.

Reply:
No. Its my mistake. I'll revert it back